client.go

```diff
@@ -53,7 +55,16 @@ type ResourceBody struct {
 // New creates a new Client
 func New(url *nurl.URL, connectorURL *nurl.URL, signer Signer, log *logrus.Entry) *Client {
 	tp := httptransport.New(url.Host, url.Path, []string{url.Scheme})
-	tp.Transport = newSigningRoundTripper(tp.Transport, signer)
+
+	if os.Getenv("GRAFTON_DEBUG") == "true" {
```

Comment:
From Slack:

> oh if you wanted to avoid the env var usage you could create alternate function signatures to use internally like
> 
> NewGraftonClient( Add new params here )
> 
> then make New call NewGraftonClient without the new debug param set

Reply:
Refactored a bit